### DISCUSSION TITLE
How to create a launch file for stella_vslam?

### DISCUSSION THREAD

**Discussion by mgrallos** [OpenVSLAM-Community/openvslam_ros#113] [Q&A] [answered]
Hi I am trying to use stella_vslam on a tello and struggling to know the nodes available on this package. can anyone provide me a general launch file for stella_vslam please?

**Comment by ymd-stella** [comment, marked as answer]
See https://stella-cv.readthedocs.io/en/latest/ros_package.html#tracking-and-mapping
